src/api/useMeasureServiceApi.tsx

```diff
@@ -96,6 +96,18 @@ export class MeasureServiceApi {
     });
   }
 
+  async deleteMeasure(measure: Measure): Promise<Response> {
+    return await axios.put(
```

Comment:
Not sure how this is working. endpoint is DELETE but this is PUT call. this should be `axios.delete`